components/Footer.js

```diff
@@ -1,7 +1,81 @@
 import React from "react";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='dark:bg-neutral-900 dark:text-gray-50 py-6'>
```

Comment:
No need to put the `dark:` that is for dark/light themes which is not something we will be doing currently, just `bg-neutral-900` is good!